examples/v3.0/api-with-examples.yaml

```diff
@@ -0,0 +1,163 @@
+openapi: "3.0.0-RC0"
+info:
+  title: Simple API overview
+  version: v2
+paths:
+  /:
+    get:
+      operationId: listVersionsv2
+      summary: List API versions
+      responses:
+        200:
+          description: |-
+            200 300 response
+          content:
+            application/json:
+              examples: 
+                - {
+                    "versions": [
+                        {
+                            "status": "CURRENT",
+                            "updated": "2011-01-21T11:33:21Z",
+                            "id": "v2.0",
+                            "links": [
+                                {
+                                    "href": "http://127.0.0.1:8774/v2/",
+                                    "rel": "self"
```

Comment:
Should we add `rel` to https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#linkObject?